tests/test_hsm.py

```diff
@@ -139,6 +139,24 @@ def test_moments_basic():
                 np.testing.assert_almost_equal(result.observed_shape.e2,
                                                distortion_2, err_msg = "- incorrect e2",
                                                decimal = decimal_shape)
+
+                # test for moments with a circular weight function
+                result_round = gal_image.FindAdaptiveMom(round_moments=True)
+                q = np.exp(-2.*math.atanh(total_shear))
+                theta = 0.5*math.atan2(g2, g1)
+                e_round = (1 - q**2)/(1 + q**2 + 2*q)
+                e1_round = e_round*math.cos(2*theta)
+                e2_round = e_round*math.sin(2*theta)
+
+                np.testing.assert_almost_equal(np.fabs(result_round.moments_sigma-sig/pixel_scale), 0.0,
+                                               err_msg = "- incorrect round dsigma", decimal = decimal)
```

Comment:
I think `dsigma` should just be `sigma` here.

Reply:
I'm happy to make the change, but all the other hsm tests use ```dismga``` when testing size.

Reply:
Huh.  I wonder why.